unitTests/ui/RegisteredNamedMethodsTest.ts

```diff
@@ -286,6 +291,18 @@ export function RegisteredNamedMethodsTest() {
         expect(setup.cmp.client instanceof NoopAnalyticsClient).not.toBeTruthy();
       });
 
+      it('should update the search interface analytics client after being re-enabled', () => {
+        RegisteredNamedMethod.disableAnalytics(setup.env.root);
+        RegisteredNamedMethod.enableAnalytics(setup.env.root);
+        expect(setup.cmp.client).toBe(setup.env.searchInterface.usageAnalytics);
```

Comment:
I don't feel like there's an actual difference about what this is testing versus line 281-284.
It seems to me that disable or enabling analytics won't actually change the result of this test.

Reply:
This test would not pass with the previous code as both client would be different (one the new Noop and the one from the SearchInterface the old Live)

Reply:
Merged both tests